RecoCTPPS/PixelLocal/python/ctppsPixelLocalReconstruction_cff.py

```diff
@@ -1,15 +1,20 @@
 import FWCore.ParameterSet.Config as cms
 
 # geometry
-#from Geometry.VeryForwardGeometry.geometryRP_cfi import *
+from Geometry.VeryForwardGeometry.geometryRP_cfi import *
 
-
-# local track fitting
+# local clusterizer
 from RecoCTPPS.PixelLocal.ctppsPixelClusters_cfi import ctppsPixelClusters
 
 # local rechit producer
 from RecoCTPPS.PixelLocal.ctppsPixelRecHits_cfi import ctppsPixelRecHits
 
+# local track producer
+from RecoCTPPS.PixelLocal.ctppsPixelLocalTracks_cfi import ctppsPixelLocalTracks
+
+from Geometry.VeryForwardGeometryBuilder.ctppsIncludeAlignments_cfi import *
+ctppsIncludeAlignments.RealFiles = cms.vstring("Alignment/CTPPS/data/RPixGeometryCorrections.xml")
```

Comment:
this apparently broke the IB:
the file path should use FileInPath instead of a plain string 
@smuzaffar @davidlange6 
If this presents a problem, I guess we'll need to revert.

the actual problem is here https://github.com/cms-sw/cmssw/blob/97fcf1127b7790836710336dd765f5ba233bb3e0/Geometry/VeryForwardGeometryBuilder/src/RPAlignmentCorrectionsDataSequence.cc#L72

Is there an easier way to discover these?

Reply:
@slava77 , I proposed a quick fix for this https://github.com/cms-sw/cmssw/pull/20708